src/runtime/manifest-parser.peg

```diff
@@ -1387,7 +1387,7 @@ EqualityExpression
   }
 
 ComparisonExpression
-  = leftExpr:AdditiveExpression tail:(whiteSpace? ('<' / '<=' / '>' / '>=') whiteSpace? AdditiveExpression)*
+  = leftExpr:AdditiveExpression tail:(whiteSpace? ('<=' / '<' / '>=' / '>') whiteSpace? AdditiveExpression)*
```

Comment:
Does this parse incorrectly? This should be slightly faster, but it should hit the '<' and then look for whiteSpace and then an AdditiveExpression (which can't start with a '=').